### DISCUSSION TITLE
A single particle suddenly flies out and exceeds a fixed boundary

### DISCUSSION THREAD

**Discussion by Xinrui157** [Anura3D/Anura3D_OpenSource#282] [Q&A] [answered]
Dear all,

I followed the tutorial step by step for the shallow foundation particles, but encountered the error:
[ERROR] : (time step: 2) Failed finding new element for particle 1 in element 1 at position 7.33333333331510 -79508.0492992555 in time step 2.
Upon checking the visualization results, I found that one particle from the foundation suddenly flew out and exceeded the boundary, while other particles seemed to behave normally. Has anyone encountered a similar issue?

<img width="905" alt="Anura3d-discussion" src="https://github.com/user-attachments/assets/5d852304-7e52-4029-a6a2-eddf6e9e6d81">

**Comment by zzuwpf** [comment]
I also encountered similar problems, some particles in the process of computing suddenly flew out of the border.

**Comment by Xinrui157** [comment, marked as answer]
I solved this problem. Please carefully set up the poisson ratio.
